### ISSUE TITLE
CentOS 7 Support

### ISSUE THREAD

**Issue by neil-fox** [pwnlandia/mhn#748]
Hi,

Is their any update on whether MHN will be be supported on CentOS 7?

Cheers

**Comment by d1str0** [comment]
This is being worked on in PR #724 

It might work, it might not. I have not tested it yet.

**Comment by neil-fox** [comment]
Yeah i tried it using a CentOS 7 iso a couple of days ago and the script fails when it comes to installing mongodb. I installed mongodb manually and it still failed.

**Comment by d1str0** [comment]
The PR worked for me up until setting firewall rules.

**Comment by neil-fox** [comment]
@d1str0 I have been attempting to install this on CentOS7 using the standard install instructions. Should i be doing something different? Thanks.

**Comment by d1str0** [comment]
You can try this branch and see if it works for you.

**Comment by neil-fox** [comment]
@d1str0 Thanks for your help.

Can confirm that MHN does work on CentOS 7.

For anybody else looking to install on CentOS 7 you will need to download MHN as normal but replace some scripts with the following updated versions:

https://github.com/pwnlandia/mhn/pull/724/files

Some awareness points for anybody else who comes across this issue:

Supervisor will need installing on CentOS 7:
- yum -y install epel-release
- yum install supervisor
- export PATH="/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/root/bin"

Make sure firewalld is installed or installation of nginx web UI will fail.

mhn-celery-worker failed for me after install, this can be sorted with the following commands:
- cd /var/log/mhn/
- chown nginx /var/log/mhn/mhn.log
- supervisorctl start mhn-celery-worker

Also be aware that once installed the deploy scripts are for Ubuntu boxes. So if your honeypots are going to be built on CentOS 7 then my understanding is that you will have to deploy the honeypots separately then add the sensors manually using the following guide:

https://github.com/pwnlandia/mhn/wiki/Incorporate-an-already-deployed-Honeypot-into-MHN

This is something I am currently having some issues with and may raise a separate case for.

**Comment by d1str0** [comment]
This has not been merged to master yet. Reopening until it is resolved.
